package.json

```
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "jquery": "^3.4.1",
-    "jquery-ui": "^1.12.1",
+    "jquery-ui": "^1.13.2",
```

The lock file shows that 1.13.3 was installed, not 1.13.2, so just for consistency.

```suggestion
    "jquery-ui": "^1.13.3",
```

---

Fixed, and reworded the commit message